applications/vessel-history/src/features/vessels/activity/vessels-activity.selectors.ts

```diff
@@ -0,0 +1,50 @@
+import { createSelector } from '@reduxjs/toolkit'
+import { RenderedEvent, selectEventsWithRenderingInfo } from 'features/vessels/activity/vessels-activity.slice';
+import { Filters, initialState, selectFilters } from './../../profile/filters/filters.slice';
```

Comment:
It's preferred to use absolute path in our imports
```suggestion
import { Filters, initialState, selectFilters } from 'features/profile/filters/filters.slice';
```